rectools/models/sasrec.py

```diff
@@ -579,11 +822,13 @@ def __init__(
         n_heads: int,
         dropout_rate: float,
         use_pos_emb: bool = True,
+        use_ids_emb: bool = True,
```

Comment:
My thoughts on interface (open for discussion):
We can't add these flags because we want to be able to pass any number of any custom net blocks. So we can't make flags for all of the blocks.

Reply:
If I understand you correctly, maybe make them optional? Since enabling and disabling the use of embeddings of item ids and category item features is a good feature in my opinion

Reply:
We can enable and disable them just by passing `item_net_block_types = [IdEmbeddingsItemNet, CatFeaturesItemNet]` or `item_net_block_types = [IdEmbeddingsItemNet]`.
This will have the same functionality of switching different net blocks on and off but it would also allow any number of custom blocks to be passed.

Reply:
Yes, I agree.